server/routes/image.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+
+const router = express.Router();
+const { deleteImages } = require("../controllers/deleteImage");
+const { multerMultiUpload, uploadImages } = require("../controllers/uploadImage");
+
+router.route("/upload").post(multerMultiUpload, uploadImages);
+router.route("/delete?").delete(deleteImages);
```

Comment:
Is there any need to add this `?` to the URL of this route?

Reply:
Not that I can think of. Went ahead and removed it. Thanks for pointing it out!